ui/src/app/views/homepage/component.css

```diff
@@ -6,30 +6,27 @@
   -o-background-size: cover;
   background-size: cover;
   height: 100%;
+  margin-left: -1rem;
+  margin-right: -0.6rem;
```

Comment:
This appears to create a strange horizontal scroll bar. Can we avoid that if possible?